server/server.go

```diff
@@ -324,6 +324,9 @@ func (s *Server) Run() {
 	r.HandleFunc("/({files:.*}).tar", s.tarHandler).Methods("GET")
 	r.HandleFunc("/({files:.*}).tar.gz", s.tarGzHandler).Methods("GET")
 
+	r.HandleFunc("/{token}/{filename}", s.headHandler).Methods("HEAD")
+	r.HandleFunc("/{action}/{token}/{filename}", s.headHandler).Methods("HEAD")
```

Comment:
please. change to `/{action:(download|get|inline)}/{token}/{filename}`
I'd rather prefer to limit the aliases